crypto/ui/ui_openssl.c

```diff
@@ -43,7 +43,7 @@
  * If unistd.h defines _POSIX_VERSION, we conclude that we are on a POSIX
  * system and have sigaction and termios.
  */
-#  if defined(_POSIX_VERSION)
+#  if defined(_POSIX_VERSION) && _POSIX_VERSION>=199309L
```

Comment:
I thought there were issues with short-circuit expressions and defined?

Reply:
No. There are [warning] issues with e,g. `#if WHATEVER>5` in cases when WHATEVER is not defined. There are real issues with e,g, `#if defined(WHATEVER) && WHATEVER(x,y)>5`, i.e. when WHATEVER is a macro **with parameters**. But there **no** issues with `#if defined(WHATEVER) && WHATEVER>5`.

Reply:
Ah, parameters; thanks.  Reconfirm.